deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -324,7 +324,23 @@ spec:
     for this version. You can get it from the releases on GitHub.
 
         curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz
-
+    
+    ### Create a KubeFed custom resource
+
+    Before joining clusters, you need to create a kubefed resource in a given namespace to drive the installation of the KubeFed.
+
+    for e.g:
+    ```
+    cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFed
+    metadata:
+      name: kubefed-resource
+    spec: 
+      scope: Namespaced 
```

Comment:
can you change this to `Cluster`? we want to encourage folks to create cluster scoped KubeFed controller than the Namespaced one. If there is a way to specify `Cluster|Namespaced` that will be good.